src/controllers/executionPlanWebviewController.ts

```diff
@@ -30,16 +29,18 @@ export class ExecutionPlanWebviewController extends ReactWebviewPanelController<
             `${xmlPlanFileName}`, // Sets the webview title
             "executionPlan",
             {
-                sqlPlanContent: executionPlanContents,
-                theme:
-                    vscode.window.activeColorTheme.kind ===
-                    vscode.ColorThemeKind.Dark
-                        ? "dark"
-                        : "light",
-                loadState: ApiStatus.Loading,
-                executionPlan: undefined,
-                executionPlanGraphs: [],
-                totalCost: 0,
+                executionPlanState: {
+                    sqlPlanContent: executionPlanContents,
+                    theme:
```

Comment:
Please use the theme variable in vscodeWebviewContextProvider. This implementation already doesn't support high contrast theme.